src/lib.rs

```diff
@@ -201,7 +203,7 @@ pub enum DependencyGroupSpecifier {
     #[serde(rename_all = "kebab-case")]
     Table {
         /// The name of the group to include
-        include_group: Option<String>,
+        include_group: String,
```

Comment:
I am not sure having an option here makes much sense?

Reply:
good call